docs/user/supported_devices.rst

```diff
@@ -316,7 +316,7 @@ mediatek-filogic
 
   - UniFi 6 Plus
 
-* ZyXEL
+* ZYXEL
```

Comment:
Wrong: 
![image](https://github.com/freifunk-gluon/gluon/assets/16539439/7b39fd3e-6716-46cb-abb3-e4dd9828e7bf)

Reply:
interesting, you are right. I had looked at their Logos mostly not continous text
[old](https://de.m.wikipedia.org/wiki/Datei:Zyxel_Logo.svg)
[new](https://commons.m.wikimedia.org/wiki/File:Z_logo_1.png#mw-jump-to-license)

Reply:
I addressed your suggestion in my latest rebase